src/main/java/seedu/address/model/Model.java

```diff
@@ -139,6 +142,7 @@
     /**
      * Adds the given person.
      * {@code person} must not already exist in the address book.
```

Comment:
@munharsha @Christopher-LM the @code person here just becareful about documentation part